solr/core/src/test/org/apache/solr/rest/TestRestManager.java

```diff
@@ -122,7 +122,7 @@ public void testRestManagerEndpoints() throws Exception {
 
   @Test
   public void testReloadFromPersistentStorage() throws IOException {
-    SolrResourceLoader loader = new SolrResourceLoader(Paths.get("./"));
+    SolrResourceLoader loader = new SolrResourceLoader(Path.of("./"));
```

Comment:
Any place we use "." or "./", it's the same as the empty string, which I think is more canonical.  Maybe a separate PR.